src/test/java/seedu/anilist/testutil/AnimeBuilder.java

```diff
@@ -15,17 +16,21 @@
 public class AnimeBuilder {
 
     public static final String DEFAULT_NAME = "Amy Bee";
```

Comment:
can change the name of Amy Bee to the most generic anime name that you can think of

Reply:
@muhammad-faruq I've changed it in my pr